src/test/java/io/radanalytics/operator/resource/YamlProcessingTest.java

```diff
@@ -83,7 +83,7 @@ public void testParseYaml1() {
 
         assertEquals(clusterInfo.getName(), "foo");
         assertEquals(clusterInfo.getWorker().getInstances().intValue(), 2);
-        assertEquals(clusterInfo.getCustomImage(), DEFAULT_SPARK_IMAGE);
+        assertEquals(clusterInfo.getCustomImage(), null);
```

Comment:
Could you pls also remove the unused import (`DEFAULT_SPARK_IMAGE`) in this class?